test/integration/debtorGroups.js

```diff
@@ -312,7 +311,7 @@ describe('(/debtor_groups) The debtor groups API', () => {
     });
 
     it('GET /debtor_groups/:uuid/invoices returns only balanced invoices for a debtor group', () => {
-      return agent.get(`/debtor_groups/${debtorGroup.uuid}/invoices?balanced=1`)
+      return agent.get(`/debtor_groups/${debtorGroup.uuid}/invoices`).query({ balanced : 0 })
```

Comment:
The test is for balanced invoices (`balanced = 1`) but you send `balanced = 0`, does it correct ?

Reply:
Hey, you are right!  Good catch.  Let me change that real quick.